packages/govuk-frontend/src/govuk/helpers/typography.test.js

```diff
@@ -54,7 +54,7 @@ describe('@mixin govuk-typography-common', () => {
         @include govuk-typography-common;
       }
       :root {
-        @include govuk-typography-common($font-family: $govuk-font-family-tabular);
+        @include govuk-typography-common($font-family: Helvetica);
```

Comment:
Feels odd to have `Helvetica` here without a matching check for the output?

I appreciate we didn't before this edit, but we did (further down) in the tabular test

```js
await expect(results).resolves.toMatchObject({
  css: expect.stringContaining('font-family: Helvetica')
})

await expect(results).resolves.toMatchObject({
  css: expect.stringContaining('font-family: "GDS Transport"')
})
```

Reply:
Not a blocker